docs/intro/getting-started-with-sphinx.rst

```diff
@@ -0,0 +1,128 @@
+Getting Started with Sphinx
+===========================
+
+Sphinx is a powerful documentation generator that
+has many great features for writing technical documentation including:
+
+* Generate web pages, printable PDFs, documents for e-readers (ePub),
+  and more all from the same sources
+* You can use reStructuredText or :ref:`Markdown <intro/getting-started-with-sphinx:Using Markdown with Sphinx>`
+  to write documentation
+* An extensive system of cross-referencing code and documentation
+* Syntax highlighted code samples
+* A vibrant ecosystem of first and third-party extensions_
+
+.. _extensions: http://www.sphinx-doc.org/en/master/ext/builtins.html#builtin-sphinx-extensions
+
+.. note::
+
+    Sphinx is a great choice for building technical documentation
+    and many projects large and small use Sphinx
+    including the Django project and the Python programming language.
+    However, Read the Docs also supports :doc:`MkDocs </intro/getting-started-with-mkdocs>`,
+    another tool for writing and building documentation.
+
+
+Quick start video
+-----------------
+
+This screencast will help you get started or you can
+:ref:`read our guide below <intro/getting-started-with-sphinx:Quick start>`.
+
+.. raw:: html
+
+    <div style="text-align: center; margin-bottom: 2em;">
+    <iframe width="100%" height="350" src="https://www.youtube.com/embed/oJsUvBQyHBs?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+    </div>
+
+
+Quick start
+-----------
+
+Assuming you have Python already, `install Sphinx`_:
+
+.. sourcecode:: bash
+
+    $ pip install sphinx
+
+Create a directory inside your project to hold your docs:
+
+.. sourcecode:: bash
+
+    $ cd /path/to/project
+    $ mkdir docs
```

Comment:
We don't mention this on the MkDocs guide

Reply:
It also creates the `docs` directory if it doesn't exist.